tools/plugins/com.liferay.ide.project.ui/src/com/liferay/ide/project/ui/modules/NewLiferayModuleProjectWizard.sdef

```diff
@@ -82,7 +82,7 @@
                      <show-label>true</show-label>
                      <child-property>Name</child-property>
                      <child-property>Value</child-property>
-                    <visible-when>${ProjectTemplateName != 'activator' &amp;&amp; ProjectTemplateName != 'servicebuilder'}</visible-when>
+                    <visible-when>${ProjectTemplateName != 'activator' &amp;&amp; ProjectTemplateName != 'servicebuilder' &amp;&amp; ProjectProvider != 'maven-module' }</visible-when>
```

Comment:
This should work fine in maven, we have support for checking this, i added it myself.  We should just fix it.